platforms/documentation/docs/src/docs/userguide/optimizing-performance/file_system_watching.adoc

```diff
@@ -48,11 +48,11 @@ Gradle supports file system watching on the following operating systems:
 
 * Windows 10, version 1709 and later
 * Linux, tested on the following distributions:
-** Ubuntu 16.04 or later
-** CentOS Stream 8 or later
-** Red Hat Enterprise Linux (RHEL) 8 or later
-** Amazon Linux 2 or later
-** Alpine Linux 3.20 or later
+** Ubuntu 16.04
+** CentOS Stream 9
+** Red Hat Enterprise Linux (RHEL) 8
+** Amazon Linux 2
+** Alpine Linux 3.20
```

Comment:
💭 I've removed the "or later" suffixes because this is supposed to be a list of specific distros we test on.